.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "dev*"
```

Comment:
We don't actually have a development branch

Reply:
I think it’s worth having that pattern as it future-proofs for when we move into a dev-test-main pattern or similar